src/SFML/Window/DRM/DRMContext.hpp

```diff
@@ -0,0 +1,190 @@
+    ~DRMContext();
+
+    ////////////////////////////////////////////////////////////
+    /// \brief Activate the context as the current target
+    ///        for rendering
+    ///
+    /// \param current Whether to make the context current or no longer current
+    ///
+    /// \return True on success, false if any error happened
+    ///
+    ////////////////////////////////////////////////////////////
+    virtual bool makeCurrent(bool current);
+
+    ////////////////////////////////////////////////////////////
+    /// \brief Display what has been rendered to the context so far
+    ///
+    ////////////////////////////////////////////////////////////
+    virtual void display();
+
+    ////////////////////////////////////////////////////////////
+    /// \brief Enable or disable vertical synchronization
+    ///
+    /// Activating vertical synchronization will limit the number
+    /// of frames displayed to the refresh rate of the monitor.
+    /// This can avoid some visual artifacts, and limit the framerate
+    /// to a good value (but not constant across different computers).
+    ///
+    /// \param enabled: True to enable v-sync, false to deactivate
+    ///
+    ////////////////////////////////////////////////////////////
+    virtual void setVerticalSyncEnabled(bool enabled);
+
+    ////////////////////////////////////////////////////////////
+    /// \brief Create the context
+    ///
+    /// \param shared       Context to share the new one with (can be NULL)
+    /// \param bitsPerPixel Pixel depth, in bits per pixel
+    /// \param settings     Creation parameters
+    ///
+    ////////////////////////////////////////////////////////////
+    void createContext(DRMContext* shared);
+
+    ////////////////////////////////////////////////////////////
+    /// \brief Create the surface
+    ////////////////////////////////////////////////////////////
+    void createSurface(unsigned int width, unsigned int height, unsigned int bpp, bool scanout);
+
+    ////////////////////////////////////////////////////////////
+    /// \brief Destroy the EGL surface
+    ///
+    /// This function must be called when the activity is stopped, or
+    /// when the orientation change.
+    ///
+    ////////////////////////////////////////////////////////////
+    void destroySurface();
+
+    ////////////////////////////////////////////////////////////
+    /// \brief Get the best EGL visual for a given set of video settings
+    ///
+    /// \param display      EGL display
+    /// \param bitsPerPixel Pixel depth, in bits per pixel
+    /// \param settings     Requested context settings
+    ///
+    /// \return The best EGL config
+    ///
+    ////////////////////////////////////////////////////////////
+    static EGLConfig getBestConfig(EGLDisplay display, unsigned int bitsPerPixel, const ContextSettings& settings);
+
+    static GlFunctionPointer getFunction(const char* name);
+
+    static drm* getDRM();
```

Comment:
Missing documentation. I'd probably also write out the short hand "getDirectRenderingManager"

Reply:
I added a brief with full name and made it protected instead. I haven't seen it written in full name anywhere else.